example/lib/main.dart

```diff
@@ -156,6 +159,43 @@ class _MyTabsState extends State<MyTabs> {
     );
   }
 
+  Card _poiCategoryInteraction() {
```

Comment:
Confirmao con producto, pero creo que en flutter queremos manter o **main.dart** o máis básico posible para os integradores

Reply:
Este código podes metelo como snippet nos test case para a validación